src/eddymotion/conftest.py

```diff
@@ -0,0 +1,54 @@
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+#
+# Copyright 2021 The NiPreps Developers <nipreps@gmail.com>
```

Comment:
In order to avoid C&P'ing headers that (seemingly) have the wrong year, how about using `2021-` e.g. if the copyright dates back to 2021 (or starting with the year NiPreps started)? I do not know how the copyright things (years, the copyright itself, etc.)

I know this is WIP and not meant to be reviewed yet, so I apologize.

Reply:
I think we should discuss this in one TechMon meeting because it's a recurring issue emerging across NiPreps. In fMRIPrep, I think we have dropped the year in the shebang and we only have it once in the package metadata.

That said, the year should be the date of creation of the copyrighted item, so technically, this should be the date of creation of the object - i.e., the year when the file was first created. This is hard to track, since often we upstream code from one place to another.

Let's chat about it and see what we all can agree :)